### ISSUE TITLE
Example environment variable in README is incorrectly formatted

### ISSUE THREAD

**Issue by SILENCE-ETERNAL** [metalmatze/transmission-exporter#29]
If I'm following the standards as listed in the [docker-compose docs](https://docs.docker.com/compose/environment-variables/#set-environment-variables-in-containers), shouldn't there be a dash in front of the variable name and then an =?

  environment:
    TRANSMISSION_ADDR: http://transmission:9091
vs
  environment:
    - TRANSMISSION_ADDR=http://transmission:9091

I'm currently having issues with the exporter connecting to my transmission daemon and I'd like to confirm I'm using the right addr/password variable format. Thanks!

**Comment by metalmatze** [comment]
No, I double checked with other docker-compose files I have and the environment is definitely an object / map with key-values.
